src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -420,7 +420,7 @@ else if (building.getTask().equals(BuildingGuardTower.Task.GUARD))
         else if (building.getTask().equals(BuildingGuardTower.Task.FOLLOW))
         {
             BlockPos pos = building.getPlayerToFollow();
-            if (pos == null || BlockPosUtil.getDistance2D(pos, building.getColony().getCenter()) > Configurations.workingRangeTownHall + Configurations.townHallPadding)
+            if (pos == null || BlockPosUtil.getDistance2D(pos, building.getColony().getCenter()) > Configurations.Gameplay.workingRangeTownHall + Configurations.Gameplay.townHallPadding)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 186 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)